assets/js/blocks/active-filters/edit.js

```diff
@@ -73,15 +73,15 @@ const Edit = ( { attributes, setAttributes } ) => {
 		);
 	};
 
-	const TagName = `h${ attributes.headingLevel }`;
+	const TagName = `h${ headingLevel }`;
 
 	return (
-		<Fragment>
+		<div className={ classNames( 'is-loading', className ) }>
```

Comment:
why `is-loading` here?

Reply:
Oh, that was a mistake, those classes should not be in the editor. :man_facepalming: 

I removed them in ea3f3acfb570fc0b410d9dd24cafd1b0d310a275.